examples/plot_simulate_evoked.py

```diff
@@ -33,10 +34,13 @@
 print(params['L2Pyr_soma*'])
 
 ###############################################################################
-# Now let's simulate the dipole
-# You can simulate multiple trials in parallel by using n_jobs > 1
+# Now let's simulate the dipole, running multiple trials in parallel
+# with JoblibBackend. It will run each trial as a separate job by default
+from hnn_core import JoblibBackend
+
 net = Network(params)
-dpls = simulate_dipole(net, n_jobs=1, n_trials=2)
+with JoblibBackend():
```

Comment:
```suggestion
with JoblibBackend(n_procs=1):
```
? is that the API?

Reply:
No, I don't think that makes sense. I just made a comment on how `n_procs` is a configuration for MPIBackend and `n_jobs` is for JoblibBackend.

Reply:
yeah okay, they are the same thing, no? You want the arguments to reflect what the original softwares call them? In any case, can you just show the user how to use it for the `JoblibBackend` so that it's documented.

```suggestion
with JoblibBackend(n_jobs=1):
```

if that's the right way to do it